dataStructures/queues/queue.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
Your Go style is very clean and follows a lot of the style guide already. I like that you combine repeated types like `priority, quantity int` instead of `priority int, quantity int`.